components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -24,6 +24,8 @@
 static const CGFloat kActionImageAlpha = (CGFloat)0.6;
 static const CGFloat kActionTextAlpha = (CGFloat)0.87;
 static const CGFloat kDividerDefaultAlpha = (CGFloat)0.12;
+/** Default edge insets for each action's image view. */
+static const UIEdgeInsets kDefaultImageEdgeInsets = (UIEdgeInsets){-16, 0, 0, 0};
```

Comment:
Do we need the last `(UIEdgeInsets)`? Can't we just put `static const UIEdgeInsets kDefaultImageEdgeInsets = {-16, 0, 0, 0};`? I'm curious what led to you adding that?

Reply:
I think they're both gross, but I think the explicit cast is less gross. If you want me to change it, I can. I felt this version was subjectively better.

Reply:
Made the change with the understanding that if you didn't care you wouldn't have commented. ;-)